src/tools/sfdk/command.cpp

```diff
@@ -1511,13 +1511,25 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
     }
 
     if (arguments.first() == "install-custom") {
-        if (!P::checkPositionalArgumentsCount(arguments, 3, 3))
+        QCommandLineParser parser;
+        parser.addPositionalArgument("name", QString(), "[name]");
+        parser.addPositionalArgument("URL|file", QString(), "[URL|file]");
+        QCommandLineOption toolingOption("tooling", QString(), "tooling");
+        parser.addOption(toolingOption);
```

Comment:
Nitpick - I would reorder it a bit and use addOptions (plural) for consistency :)

```suggestion
        QCommandLineParser parser;
        QCommandLineOption toolingOption("tooling", QString(), "tooling");

        parser.addOptions({toolingOption});
        parser.addPositionalArgument("name", QString(), "[name]");
        parser.addPositionalArgument("URL|file", QString(), "[URL|file]");
```